<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLineProfile</class>
 <widget class="QDialog" name="DialogLineProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Line Profile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Resolution</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditResolution">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Sample points</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditSamplePoints">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Iso-line 0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxIsoLine1">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Iso-line 1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxIsoLine2"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCompute">
       <property name="text">
        <string>Compute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExport">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelError">
     <property name="styleSheet">
      <string notr="true">color:red;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBoxIsoLine1</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DialogLineProfile</receiver>
   <slot>OnComboIsoLine(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxIsoLine2</sender>
   <signal>activated(int)</signal>
   <receiver>DialogLineProfile</receiver>
   <slot>OnComboIsoLine(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCompute</sender>
   <signal>clicked()</signal>
   <receiver>DialogLineProfile</receiver>
   <slot>OnCompute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonExport</sender>
   <signal>clicked()</signal>
   <receiver>DialogLineProfile</receiver>
   <slot>OnExport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnComboIsoLine(int)</slot>
  <slot>OnCompute()</slot>
  <slot>OnExport()</slot>
 </slots>
</ui>
